Readme_template.md

```diff
@@ -39,4 +59,21 @@ After successfull installation run:
 ```bash
 ./validate.sh -is
 ```
-See `../validation_results/` for results.
\ No newline at end of file
+See `../validation_results/` for results.
+ 
+## Usage
+See [Usage.md](./USAGE.md)
+ 
+## Contributing
+If you want to contribute, please check our [CONTRIBUTING.md](./CONTRIBUTING.md).
+ 
+## License
+ 
+Copyright [yyyy] gematik GmbH
```

Comment:
wer setzt die  Jahreszahl?

Reply:
Im README.md habe ich folgende Zeile hinzugefügt
* Set the current year in the LICENSE file (bottom of the page) AND the Readme_template.md